src/components/organisms/AuthenticationModal/InitialForm/InitialForm.scss

```diff
@@ -1,22 +1,13 @@
 @import "scss/constants.scss";
 
 .initial-form {
-  .create-account-button {
-    box-shadow: inset 0 0 0 3px $primary !important;
-    background-color: $primary !important;
-    color: white !important;
-
-    &:hover {
-      background-color: darken($color: $primary, $amount: 20%) !important;
-    }
-  }
```

Comment:
Instead of deleting this, it probably would have been better to comment it out + add an `@debt` comment.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1664/commits/3a4c26a0b571d241b6017166ceeea7fb272e0654